lib/server/swarm.js

```diff
@@ -29,15 +29,7 @@ function Swarm (infoHash, server) {
       peer_id: peer.peerId
     }
     self._onAnnounceStopped(params, peer, peer.peerId)
-
-    // When a websocket peer is evicted, and it's not in any other swarms, close
-    // the websocket to conserve server resources.
-    if (peer.socket && peer.socket.infoHashes.length === 0) {
-      try {
-        peer.socket.close()
-        peer.socket = null
-      } catch (err) {}
-    }
+    peer.socket = null
```

Comment:
Why not use delete? I think we've discussed it before but I couldn't recall the reason.